### PR TITLE
CLN: misplaced TimedeltaIndex tests

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

**Review by WillAyd — APPROVED** [review]
lgtm. similar comment on other PR probably best to just do a straight move and do new tests / parametrization separately to make reviews easier, but I think this one is OK

File: pandas/tests/indexes/timedeltas/test_tools.py [hunk 1]
```diff
@@ -57,6 +57,17 @@ def test_to_timedelta(self):
...
 
+    def test_to_timedelta_dataframe(self):
+        # GH 11776
+        arr = np.arange(10).reshape(2, 5)
+        df = pd.DataFrame(np.arange(10).reshape(2, 5))
+        for arg in (arr, df):
+            with pytest.raises(TypeError, match="1-d array"):
+                to_timedelta(arg)
+            for errors in ["ignore", "raise", "coerce"]:
+                with pytest.raises(TypeError, match="1-d array"):
```
Comment (jreback) [inline pandas/tests/indexes/timedeltas/test_tools.py]: shouldn't this actually have different behavor for coerce / ignore? e.g. this should *not* raise.

Comment (jbrockmendel) [reply pandas/tests/indexes/timedeltas/test_tools.py]: with a 2D input it always raises

Comment (jreback) [reply pandas/tests/indexes/timedeltas/test_tools.py]: ahh ok, its to_timedelta, nvm

**Pull request merged**